src/bentoml/_internal/utils/__init__.py

```diff
@@ -48,10 +48,29 @@
 
 C = t.TypeVar("C")
 T = t.TypeVar("T")
-_T_co = t.TypeVar("_T_co", covariant=True, bound=t.Any)
 
 rich_console = Console(theme=None)
 
+# lightweight port from psutil._common.py to avoid circular dependency in bazel
```

Comment:
Sorry but I think it ridiculous that introducing a build tool requires project dependency changes.

Reply:
There should be some other ways to do that with bazel right?

Reply:
Actually, it is not because of bazel. I ran into the issue before because of a bug on my end. Let me refactor the comment.